### PR TITLE
Feat eth oracles ac updates

### PR DESCRIPTION
**Pull request by MuthuVega**
Updates to the acceptance criteria to match implementation

### CODE REVIEW

**Review by AnExsomnis — DISMISSED** [review]

**Review by pscott31 — DISMISSED** [review]

**Review by gordsport — CHANGES_REQUESTED** [review]
@MuthuVega 

Please can you update the protocol/features.json file with the additions/removals

File: protocol/0082-ETHD-ethereum-data-source.md [hunk 1]
```diff
@@ -181,9 +177,8 @@ Select {
 
 ### Protocol Upgrade
 
-1. Create / amend an external oracle data source and before it is enacted perform a protocol upgrade. The oracle data source should be enacted at the correct time after the upgrade (<a name="0082-ETHD-042" href="#0082-ETHD-042">0082-ETHD-042</a>)
-2. Create / amend an external oracle data source with enactment time that falls during a protocol upgrade. The oracle data source should be enacted immediately after the network is up after the protocol upgrade (<a name="0082-ETHD-043" href="#0082-ETHD-043">0082-ETHD-043</a>)
-3. Phase 2 - Have a network running g with a mix of internal and external active and inactive oracles. Perform a protocol upgrade. Once the network is up , the state of the various oracles should be the same as before the protocol upgrade and either catch up all missed events or start processing new events based on config (<a name="0082-ETHD-044" href="#0082-ETHD-044">0082-ETHD-044</a>)
+1. Protocol upgrade should have no impact on a market using an ethereum oracle for settlement and/or termination (<a name="0082-ETHD-042" href="#0082-ETHD-042">0082-ETHD-042</a>)
```
Comment (Jiajia-Cui) [inline protocol/0082-ETHD-ethereum-data-source.md]: more specific wording 

more than 1 AC

@jgsbennett @vega-paul

File: protocol/0082-ETHD-ethereum-data-source.md [hunk 2]
```diff
@@ -111,15 +111,11 @@ Select {
...
-2. Amend an existing ethereum data source and change the contract address and enact the proposal. Once enacted, the data should be sourced from the new smart contract. Try amending the other fields in the oracle data source and the changes should take effect after the proposals are enacted (<a name="0082-ETHD-006" href="#0082-ETHD-006">0082-ETHD-006</a>)
-3. Phase 2 - Amend an existing ethereum data source and change the events that we are listening to and enact the proposal. Once enacted , the data should be sourced from the amended events. Try amending the other fields in the oracle data source and the changes should take effect after the proposals are enacted (<a name="0082-ETHD-007" href="#0082-ETHD-007">0082-ETHD-007</a>)
+1. Update an existing market using the market update proposal to change the smart contract address and read method. The changes take effect after the market update proposal is enacted and data is sourced from the new smart contract. (<a name="0082-ETHD-005" href="#0082-ETHD-005">0082-ETHD-005</a>)
```
Comment (Jiajia-Cui) [inline protocol/0082-ETHD-ethereum-data-source.md:117]: do we have enough coverage?

more ACs 

@MuthuVega

**Comment by MuthuVega** [comment]
Closing this PR as opened a new one - https://github.com/vegaprotocol/specs/pull/1875

**Pull request closed without merging**